### PR TITLE
clarification about global reader

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#847

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/add-users/about-admin-roles.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ You'll probably only need to assign the following roles in your organization. (F
 |Exchange admin     |  Full access to Exchange Online, creates and manages groups, manages service requests, and monitors service health.    |
 |External identity provider admin    |     Configure identity providers for use in direct federation.    |
 |Global admin     |    Has unlimited access to all management features and most data in all admin centers.     |
-|Global reader     |    Has read-only access to all management features and most data in all admin centers.      |
+|Global reader     |    Has read-only access to all management features and most data in admin centers. For detailed description of access rights and limitations of this role, please see [Administrator role permissions in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/users-groups-roles/directory-assign-admin-roles#global-reader)     |
```
Comment (JohanFreelancer9) [inline Office365-Admin/add-users/about-admin-roles.md]:
```suggestion
|Global reader     |    Has read-only access to all management features and most data in admin centers. For a detailed description of access rights and limitations of this role, please see [Administrator role permissions in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/users-groups-roles/directory-assign-admin-roles#global-reader).    |
```

**Comment by mypil** [comment]
@MaratMussabekov -- please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@kwekuako - PR has been copy edited and is ready for final review. This is based on issue #847

Thank you.

cc: @kenwith

**Pull request merged**